tests/unit/test_docs/test_api.py

```diff
@@ -120,3 +121,12 @@ def test_tutorial007(app_runner, mocker: MockerFixture):
     time_mock = mocker.patch.object(Api, 'insert_data')
     app_runner(tutorial007.scheduled_app, time_event)
     time_mock.assert_called_once()
+
+
+def test_tutorial008(app_runner, mocker: MockerFixture):
+    time_event = ScheduledDataTimeEvent(
+        asset_id=0, company_id=0, start_time=0, end_time=0
+    )
+    time_mock = mocker.patch.object(Api, 'insert_data')
+    app_runner(tutorial008.scheduled_app, time_event)
+    time_mock.assert_not_called()
```

Comment:
```suggestion
    app_runner(tutorial008.scheduled_app, time_event)
```

Reply:
This [is addressed](https://github.com/corva-ai/python-sdk/pull/81/commits/fe5a3dfda5454d40892c4e5d386114ec1db2943c).